.github/workflows/release.yml

```diff
@@ -15,6 +15,7 @@ jobs:
       - name: Setup dependencies
         uses: ./.github/actions/setup-dependencies
 
-      - run: poetry publish --build
+      - run: uv build
+      - run: uv publish
         env:
-          POETRY_PYPI_TOKEN_PYPI: ${{secrets.POETRY_PYPI_TOKEN_PYPI}}
+          UV_PUBLISH_TOKEN: ${{secrets.POETRY_PYPI_TOKEN_PYPI}}
```

Comment:
Should also update the secret...

Reply:
It would be nice to the secret having a descriptive name, e.g. `PYPI_UPLOAD_TOKEN` or something, but I have no authority over our pypi mirror.

@martonilles should be able to set-up a new token